plugins/lsp/language-servers/default.nix

```diff
@@ -20,6 +20,12 @@ with lib; let
       description = "Enable bashls, for bash.";
       package = pkgs.nodePackages.bash-language-server;
     }
+    {
+      name = "biome";
+      description = "Enable Biome, Toolchain of the Web";
+      package = pkgs.biome;
+      cmd = cfg: ["${cfg.package}/bin/biome" "lsp-proxy"];
```

Comment:
I don't think that this is needed as `nvim-lspconfig` has set `biome lsp-proxy` as the default command.
As `cfg.package` will be added to `extraPackages` (and thus `"${cfg.package}/bin"` will be added to the `$PATH`), the command should resolve nicely.